GliaWidgets/Sources/Interactor/Interactor.swift

```diff
@@ -65,7 +65,7 @@ class Interactor {
 
     private var observers = [() -> (AnyObject?, EventHandler)]()
 
-    var state: InteractorState = .none {
+    @Published var state: InteractorState = .none {
```

Comment:
Does this change mean that `didSet` below should be removed and reworked in favour of handling of `state` change by subscription?

Reply:
I've created a ticket for migrating to subscriptions